src/clostack/client.clj

```diff
@@ -10,12 +10,12 @@
 (defn http-client
   "Create an HTTP client"
   ([]
-   http-client {})
-  ([{:keys [config pool]}]
+   (http-client {}))
+  ([{:keys [config pool opts] :or {opts {}}}]
    {:config (or config (config/init))
     :opts (if (some? pool)
-              {:pool pool}
-              {})}))
+            (assoc opts {:pool pool})
+            opts)}))
```

Comment:
then, drop the `:pool` and keep only `:opts`

Reply:
Done ;)